azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participant/grid/cell/ParticipantGridCellAvatarView.kt

```diff
@@ -59,6 +63,29 @@ internal class ParticipantGridCellAvatarView(
         }
     }
 
+    fun updatePersonaData() {
+        val personaData =
+            getPersonaDataCallback(participantViewModel.getParticipantUserIdentifier())
+        if (personaData == null) {
+            // force bitmap update be setting resource to 0
+            avatarView.setImageResource(0)
+            setDisplayName(participantViewModel.getDisplayNameStateFlow().value)
+        } else {
+            if (lastPersonaData != personaData) {
```

Comment:
Could we try 'else if' conditional block here i.e. else if (personalData != lastPersonaData) so that we don't have a conditional block within another conditional block?